code/data_structures/src/linked_list/README.md

```diff
@@ -0,0 +1,23 @@
+# Linked Lists
+Description
+---
```

Comment:
While this syntax for headers work, could you use the `##` syntax, which is more common as well?

Reply:
added ```##``` for h2 level heading